docker-compose.build.yml

```diff
@@ -120,13 +120,13 @@ services:
   ########################################################################
 
   pulumi:
-    image: ${REGISTRY:-grapl}/local-pulumi:${TAG:-latest}
+    image: ${DOCKER_REGISTRY:-grapl}/local-pulumi:${TAG:-latest}
     build:
       context: .
       dockerfile: Dockerfile.pulumi
 
   provisioner:
-    image: ${REGISTRY:-grapl}/provisioner:${TAG:-latest}
+    image: ${DOCKER_REGISTRY:-grapl}/provisioner:${TAG:-latest}
```

Comment:
What is the benefit of including a default value in all these (the "grapl" bit)? It seems that since the `localhost:5000` value is now in `local-grapl.env` that covers the Local Grapl case. Since we use Cloudsmith for real registry operations, we will always have to set `DOCKER_REGISTRY` accordingly when we're doing stuff in CI. What would the usecase be for defaulting to "grapl" (and thus Docker Hub, which we aren't using)?

Reply:
I didn't want to break anything, basically. it used to say `grapl` and as I was changing it, I wanted to avoid breaking anything while we're in a transition phase.

Reply:
I think it should be safe, but we can also loop back around later if you like 👍